cocos/network/WebSocket.h

```diff
@@ -238,7 +239,7 @@ class CC_DLL WebSocket
 
     struct lws*         _wsInstance;
     struct lws_context* _wsContext;
-    std::shared_ptr<bool> _isDestroyed;
+    std::shared_ptr<std::atomic<bool>> _isDestroyed;
```

Comment:
you could only capture by `this`, but then it's a reference capture rather than an assignment capture.